### PR TITLE
86 update get involved section first half

### PR DESCRIPTION
**Pull request by greenhacks**
# Pull Request Process

## Describe the issue you are trying to resolve

This PR addresses six of 12 tasks in Issue 86 - (https://github.com/specollective/spec-frontend/issues/86) - am trying to merge in key changes as they are made. 

## Describe the changes you've made to resolve the issue
- Modifies the `GetInvolved`, `ContactForm`, and `DonationTable` components  to address the following issues (taken verbatim from linked issue above)

> - Use the contribution tiers as the primary image 
> - Remove learn more show more info interaction
> - Link to documentation site (we can choose the specific page later)
> - Body copy font is incorrect, should be Montserrat
> - Donate button should open OC in a new window
> - Contact Us button doesn't anchor to the Contact form at the bottom of the page

### Before and After - cards
font-before
montserrat-font-after

### Before and After - contribution table
1) contribution-tier-before
2) contribution-table-before-2

After: contribution-tier-after

### Before and After - mobile (IPhone XR)
mobile-before
mobile-after

## Additional Notes
- On the contribution table in `GetInvolved.tsx`, I will work to right align the ellipsis (`...`) on the next PR per Figma design.
- For mobile, I chose to keep the existing `DonationImg` image since that adding a contribution tier would have likely necessitated another card. If another card with the contribution tier is desired, perhaps that could be a future PR.

## Code Review Checklist

- [ ] Have the package.json or package-lock.json been changed? If so, does the pull request description say why?
- [ ] Has the UI for mobile and desktop been changed, did you test on IphoneXr and desktop views?
- [ ] Does the UI match the requirements in the ticket and designs in figma; does it pass the visual smell test?
- [ ] Have TypeScript files been changed, does the TypeScript pass the smell test?
- [ ] If you don't feel super confident in your review, did you assign someone more senior to double check?

### CODE REVIEW

File: components/DonationTable.tsx [hunk 1]
```diff
@@ -6,50 +6,49 @@ import { tableContents } from '../constants/table-of-contents'
 function DonationTable() {
   return (
       <div>
```
Comment (najwanan) [inline components/DonationTable.tsx:8]: If you want to achieve a large table add a tailwind class on line 8, where the div has no current classes for stylind add add a width of 4/5 or something similar i.e. className="w-4/5"

File: components/GetInvolved.tsx [hunk 2]
```diff
@@ -108,70 +108,56 @@ export default function GetInvolved() {
...
-								<div className="flex justify-end" data-closeable>
-									<button
-										type="button"
-										onClick={() => setDonationTable("Image")}>
-										<span aria-hidden="true" className="">
-											&times;
-										</span>
-									</button>
-                  </div>
+						<div id="donation-table">
```
Comment (najwanan) [inline components/GetInvolved.tsx:156]: Here you want to add a className="lg:w-1/2" to the div with the id "donation-table"
In the div directly underneath (that wraps the donation table component" add this className="flex justify-center lg:w-full"
You'll still have to adjust the left margins on the left half of get involved section but will get you closer to the figma

**Review by najwanan — APPROVED** [review]
Looks good to me. I left some comments about how to expand the table if you want. You can leave as is, merge and tackle that on the next round of changes or change now. It's up to you! :-)

**Comment by greenhacks** [comment]
Thank you, @najwanan! I will go ahead and merge in for now, but I would like to incorporate your suggestions on the next PR.

**Pull request merged**